src/include/gdb_packet.h

```diff
@@ -34,17 +34,25 @@
 #define GDB_PACKET_NOTIFICATION_START '%'
 #define GDB_PACKET_ESCAPE_XOR         (0x20U)
 
+#if defined(__MINGW32__) || defined(__MINGW64__) || defined(__CYGWIN__)
+#define GDB_DEBUG_FORMAT_ATTR __attribute__((format(__MINGW_PRINTF_FORMAT, 1, 2)))
```

Comment:
Given where this is used, the `_DEBUG` part can be dropped - the functions below aren't debugging functions but rather are used to tell the user about goings on via their GDB connection.